test/modules/navigator.js

```diff
@@ -41,6 +41,15 @@
             }
 
             resetTestVariables();
+
+            // if ( viewer && viewer.close ) {
```

Comment:
We can clean out this commented code.

Reply:
Cleaned up along with the rest of the similar tests